src/Installers/Windows/Wix.targets

```diff
@@ -7,7 +7,7 @@
 
   <PropertyGroup>
     <GuidInputs>$(Version);$(Platform)</GuidInputs>
-    <GuidInputs Condition="'$(IsFinalBuild)' != 'true'">$(GuidInputs);$(BuildNumber)</GuidInputs>
+    <GuidInputs Condition="'$(IsFinalBuild)' != 'true'">$(GuidInputs);$(BuildNumber);$(BuildNumber)</GuidInputs>
```

Comment:
Adding `$(BuildNumber)` again changes nothing. Need to make use of that property unconditional.

Reply:
Ah, whoops, somehow missed it was already there. Revisiting this now